public/includes/find-house.php

```diff
@@ -0,0 +1,51 @@
+<?php
+require_once('db.php');
+
+function get_states() {
+    $mysqli = db_connection();
+    $sql_query = "SELECT * FROM `states`";
+    $result = mysqli_query($mysqli,$sql_query);
+    $json_response = array();
+    while ($row = mysqli_fetch_assoc($result)) {
+        $json_response[] = $row;
+    }
+    echo json_encode($json_response);
```

Comment:
It is preferable to return the HTTP status for success and error states while returning the data. A rough search got me this - https://www.codeofaninja.com/2017/02/create-simple-rest-api-in-php.html (Please take note of the function used - http_response_code(400))

Reply:
Can you try to return the response instead of echoing it 🤷‍♂️